docs/react-v9/contributing/testing-with-jest.md

```diff
@@ -0,0 +1,234 @@
+> 🚨🚨 **This page is primarily about `@fluentui/react-components` ("v9") and `@fluentui/react` ("v8") and related packages.** 🚨🚨&nbsp; [See this page for `@fluentui/react-northstar` ("v0").](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/test-a-feature.md)
+
+## Overview
+
+Fluent UI's unit, functional, and snapshot tests are built using [Jest](https://jestjs.io/). This allows us to run tests in a Node environment but simulates the browser using [jsdom](https://github.com/jsdom/jsdom). ([See this page for other types of tests.](Testing))
+
+We use various libraries on top of Jest for rendering and interacting with React components:
+
+- New tests should use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro)
+- Legacy tests may also use:
+  - [Enzyme](https://enzymejs.github.io/enzyme/) -- **not allowed in new tests**
+  - [react-test-renderer](https://reactjs.org/docs/test-renderer.html), primarily for snapshot testing _(snapshots are discouraged in new tests)_
+
+## Running tests
+
+Our Jest setup generally require that packages be built before testing, so before running tests the first time, you should run `yarn build --to my-package` from the repo root.
+
+To run the tests for one package, `cd` to that package and then run `yarn test`.
+
+To run an individual test (or technically any tests with a path matching this substring), `cd` to the relevant package and run `yarn jest MyTestName`.
+
+(You can also run tests for the whole repo by running `yarn test` at the repo root. This will build beforehand if needed.)
+
+## Running tests in watch mode
+
+When you are developing tests, use the watch mode to run the tests as you write them!
+
+1. Go to the package folder where you want to run the tests.
+2. The start command varies by library: for v8, `yarn start-test`, or for v9, `yarn test --watch`
+3. Edit and saving tests should now cause the console to re-run the tests you have added/modified.
+
+### Debugging
+
+The repo includes launch configurations for debugging tests using Visual Studio Code. (You could also configure debugging in another editor of your choice.)
+
+1. Set breakpoints in the test file (`*.test.ts` or `*.test.tsx`)
+2. Open the **Run** (debugger) pane in the sidebar and choose the configuration you want: usually **Debug current open test** to run only the current open test, or **Debug test** to run all tests for the package the current file is in
+3. Start debugging
+
+## Jest basics
+
+Tests in Jest are written similarly to Mocha tests, though Jest includes a number of assertions that work similarly to Chai. (If you've never used either of those, don't worry about it!)
```

Comment:
I'm not sure any of these information is necessary as this document doesn't provide info about what is mocha nor chai. 

Lets simplify, make it concise and to the point.

```suggestion
```